interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,67 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+
+//Function do translate from another language to English
+if (!(function_exists('ltox'))) {
+
+    /**
+     * @param $wordInLang
+     * @param bool $oneWord
+     * @return bool or English text of var
+     */
+    function ltox( $wordInLang, $oneWord = true )
```

Comment:
What does the $oneWord flag do?

Reply:
I'm propose this flag for future using, when we are need search by phrase in another languages